src/validation.ts

```diff
@@ -0,0 +1,36 @@
+import {Request, Response} from 'express';
+// eslint-disable-next-line node/no-unpublished-import
+import Joi from 'joi';
+
+export const userSchema = {
+  query: Joi.object({
+    loginSubstring: Joi.string(),
+    limit: Joi.number().integer().greater(0),
+  }),
+  body: Joi.object({
+    id: Joi.string(),
```

Comment:
Remove it. Backend has to generate id of the user. And you can't update it

Reply:
created for put separate schema with id